### ISSUE TITLE
Feedback target results page heading levels inconsistent

### ISSUE THREAD

**Issue by HRemonen** [UniversityOfHelsinkiCS/palaute#1317]
The feedback target results page has inconsistent use of the heading levels.
The heading levels for "Monivalintakysymykset", "avoimet kysymykset", etc. should be of heading level 2 not 4. Now heading level 3 is skipped but also the headings are not supposed to be subheadings for the previous heading level 2.

![Image](https://github.com/UniversityOfHelsinkiCS/palaute/assets/81435393/9a471a96-4425-495d-83a0-09f22a556e9c)

## How to fix
- Change the heading elements to h2 and style accordingly. Heading levels should not be used to just style elements.
